app/models/katello/flatpak_remote.rb

```diff
@@ -0,0 +1,13 @@
+module Katello
+  class FlatpakRemote < Katello::Model
+    has_many :katello_flatpak_remote_repositories, dependent: :destroy, class_name: 'Katello::FlatpakRemoteRepository'
+    has_many :katello_flatpak_remote_repository_manifests, through: :katello_flatpak_remote_repositories, source: :katello_flatpak_remote_repository_manifests
+    belongs_to :organization, inverse_of: :flatpak_remotes
+
+    alias_attribute :repositories, :katello_flatpak_remote_repositories
```

Comment:
Do you think this alias should be `remote_repositories` instead of `repositories` in case we end up with some relation to real `::Katello::Repository` records in the future?

Reply:
Actually, the name could just be set to remote_repositories even in the has_many call. I don't think it needs to match the DB table name if the class name is passed in too.